client/purplecaffeine/core.py

```diff
@@ -343,10 +389,8 @@ def save(self, trial: Trial) -> str:
         Returns:
             self.path: path of the trial file
         """
-        trial_id = trial.name + datetime.now().strftime("%Y%m%d%H")
-        with open(
-            os.path.join(self.path, trial_id + ".json"), "w", encoding="utf-8"
-        ) as trial_file:
+        save_path = os.path.join(self.path, f"{trial.uuid}.json")
```

Comment:
In local, was more logical to me to have file we can understand what they are by their name

Reply:
good point. Maybe we can move files in this structure `<path>/<experiment_name>/<uuid>.json`

Reply:
ah, then to load it back we will need to provide name and uuid =(

Reply:
Yeah, we don't have simple answer here :/